changelogs/fragments/dropping-ansible29.yml

```diff
@@ -0,0 +1,2 @@
+major_changes:
+  - "Dropping support for Ansible 2.9, ansible-core 2.14 will be minimum required version for this release"
```

Comment:
So the next release will be 2.0.0 since this is a breaking change for all users of ansible-core < 2.14?

Reply:
Looks like it needs to be, yes.